beacon_node/network/src/router.rs

```diff
@@ -560,8 +559,11 @@ impl<T: BeaconChainTypes> Router<T> {
     fn send_status(&mut self, peer_id: PeerId) {
         let status_message = status_message(&self.chain);
         debug!(self.log, "Sending Status Request"; "peer" => %peer_id, &status_message);
-        self.network
-            .send_processor_request(peer_id, RequestType::Status(status_message));
+        self.network.inform_network(NetworkMessage::SendRequest {
+            peer_id,
+            request_id: SyncRequestId::Status,
+            request: RequestType::Status(status_message),
```

Comment:
This has changed a status message type to be related to sync, I think.

Reply:
but in this case we don't care if it's `Router` or `Sync` right?